modular_skyrat/modules/sec_haul/code/technode/all_nodes.dm

```diff
@@ -7,67 +7,47 @@
 	description = "In the wake of the NRI Border Conflict, there was a drive to advances our armament, learn how sol does it."
 	prereq_ids = list("exotic_ammo")
 	design_ids = list(
-		"sol40_riflstandardemag",
 		"solgrenade_extmag",
 		"s12g_slug",
-		"s12c_antitide",
 		"s12g_buckshot"
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = TECHWEB_TIER_2_POINTS)
 
+/datum/design/kiboko_box_mag
+	name = "Kiboko Grenade Box Magazine"
+	desc = "An extended capacity box magazine for compatible grenade launcher."
```

Comment:
Did you add this in this PR or was it moved?

Reply:
Moved this